contracts/RiskSteward/MarketCapsRiskSteward.sol

```diff
@@ -0,0 +1,177 @@
+import { AccessControlledV8 } from "../Governance/AccessControlledV8.sol";
+import { IRiskSteward } from "./IRiskSteward.sol";
+
+/**
+ * @title MarketCapsRiskSteward
+ * @author Venus
+ * @notice Contract that can update supply and borrow caps received from RiskStewardReceiver
+ * @custom:security-contact https://github.com/VenusProtocol/governance-contracts#discussion
+ */
+contract MarketCapsRiskSteward is IRiskSteward, Initializable, Ownable2StepUpgradeable, AccessControlledV8 {
+    /**
+     * @notice The max increase bps for the update relative to the current value
+     */
+    uint256 public maxIncreaseBps;
+
+    /**
+     * @notice Address of the CorePool comptroller used for selecting the correct comptroller abi
+     */
+    ICorePoolComptroller public immutable CORE_POOL_COMPTROLLER;
+    /**
+     * @notice Emitted when a supply cap is updated
+     */
+    event SupplyCapUpdated(address market, uint256 newSupplyCap);
+
+    /**
+     * @notice Emitted when a borrow cap is updated
+     */
+    event BorrowCapUpdated(address market, uint256 newBorrowCap);
+
+    /**
+     * @notice Emitted when the max increase bps is updated
+     */
+    event MaxIncreaseBpsUpdated(uint256 oldMaxIncreaseBps, uint256 newMaxIncreaseBps);
+
+    /**
+     * @notice Thrown when a maxIncreaseBps value of 0 is set
+     */
+    error InvalidMaxIncreaseBps();
+
+    /**
+     * @notice Thrown when an updateType that is not supported is operated on
+     */
+    error UnsupportedUpdateType();
+
+    /**
+     * @notice Thrown when the new value of an update is our of range
+     */
+    error UpdateNotInRange();
+
+    /// @custom:oz-upgrades-unsafe-allow constructor
+    constructor(address corePoolComptroller_) {
+        require(corePoolComptroller_ != address(0), "Core Pool Comptroller address must not be zero");
+        CORE_POOL_COMPTROLLER = ICorePoolComptroller(corePoolComptroller_);
+        _disableInitializers();
+    }
+
+    function initialize(address accessControlManager_, uint256 maxIncreaseBps_) external initializer {
+        __Ownable2Step_init();
+        __AccessControlled_init_unchained(accessControlManager_);
+        if (maxIncreaseBps_ == 0) {
+            revert InvalidMaxIncreaseBps();
+        }
+        maxIncreaseBps = maxIncreaseBps_;
+    }
+
+    function setMaxIncreaseBps(uint256 maxIncreaseBps_) external {
+        _checkAccessAllowed("setMaxIncreaseBps(uint256)");
+        uint256 _maxIncreaseBps = maxIncreaseBps;
+        if (maxIncreaseBps_ == 0) {
+            revert InvalidMaxIncreaseBps();
+        }
+        maxIncreaseBps = maxIncreaseBps_;
+        emit MaxIncreaseBpsUpdated(_maxIncreaseBps, maxIncreaseBps_);
+    }
+
+    /**
+     * @notice
+     * @param update
+     */
+    function processUpdate(RiskParameterUpdate calldata update) external {
+        _checkAccessAllowed("processUpdate(RiskParameterUpdate)");
```

Comment:
Would it make sense to only accept calls from the `RiskStewardReceiver`, instead of using the ACM? Or do we plan to call this function from a different wallet?